libcontainer/error.go

```diff
@@ -46,6 +46,8 @@ func (c ErrorCode) String() string {
 		return "Container is not running"
 	case ConsoleExists:
 		return "Console exists for process"
+	case ProcessNotExecuted:
+		return "No process operations"
```

Comment:
The name and message seems not well matched, can we change `ProcessNotExecuted` to something like `NoProcessOps`? The name `ProcessNotExecuted` seems has more meaning than on ops.